_config.yml

```diff
@@ -134,6 +134,7 @@ highlighter: rouge
 permalink: /:categories/:title.html:output_ext
 baseurl:
 scala3ref: "https://dotty.epfl.ch/docs/reference"
+scaladocUsage: "https://dotty.epfl.ch/docs/usage/scaladoc"
```

Comment:
If we only use this URL on this one page, then I would rather expand it below. This is a global config file -- we just have `scala3ref` here because we were afraid the URL structure will change and didn't want to go through all of the files in this repo :)

Reply:
Changed